package.json

```diff
@@ -6,16 +6,16 @@
     "eslint-check": "eslint \"{src,test}/**/*.ts\"",
     "eslint-fix": "eslint \"{src,test}/**/*.ts\" --fix",
     "build": "tsc -p tsconfig-build.json --skipLibCheck && cp -R src/services/price/protos lib/services/price/ && cp -R src/services/dealer-price/proto lib/services/dealer-price/ && tscpaths -p tsconfig.json -s ./src -o ./lib",
-    "trigger": "yarn build && source ./scripts/export-local.sh && node lib/servers/trigger.js | pino-pretty -c -l",
+    "trigger": "yarn build && node lib/servers/trigger.js | pino-pretty -c -l",
     "watch": "nodemon -V -e ts,graphql -w ./src -x yarn run start",
     "watch-trigger": "nodemon -V -e ts,graphql -w ./src -x yarn trigger",
     "kill-graphql": "kill $(lsof -t -i:4001) & kill $(lsof -t -i:4002)",
     "forward-mongodb": "kubectl port-forward $(kubectl get pods -l app.kubernetes.io/name=mongodb | awk '/mongo/ {print $1}') 27017",
-    "cron": "yarn build && source scripts/export-local.sh && node src/servers/cron.js",
+    "cron": ". ./.envrc && yarn build && node lib/servers/cron.js",
     "exporter": "yarn build && node lib/servers/exporter.js",
-    "lnd-connection": "source scripts/export-local.sh && echo $MACAROON | base64 -d > ./macaroon && echo $TLS | base64 -d > ./tls.cert && lncli --network regtest --macaroonpath=./macaroon --tlscertpath=./tls.cert --rpcserver=${LNDDNS}:${LNDRPCPORT} getinfo",
-    "lncli": "source scripts/export-local.sh && echo $MACAROON | base64 -d > ./macaroon && echo $TLS | base64 -d > ./tls.cert && lncli --network regtest --macaroonpath=./macaroon --tlscertpath=./tls.cert --rpcserver=${LNDDNS}:${LNDRPCPORT}",
-    "bitcoin-cli": "source ./scripts/export-local.sh && kubectl exec bitcoind-0 -- bitcoin-cli -rpcuser=rpcuser -rpcpassword=$BITCOINDRPCPASS -regtest",
+    "lncli1": "lncli --network regtest --macaroonpath=./dev/lnd/lnd1.macaroon --tlscertpath=./dev/lnd/tls.cert --rpcserver=${LND1_DNS}:${LND1_RPCPORT}",
+    "lncli2": "lncli --network regtest --macaroonpath=./dev/lnd/lnd2.macaroon --tlscertpath=./dev/lnd/tls.cert --rpcserver=${LND2_DNS}:${LND2_RPCPORT}",
```

Comment:
I thought I left a comment here about moving these scripts to `bin` since they have no dependency on nodejs so running them via `yarn` has disadvantages.
But it perhaps I forgot to submit it.

Reply:
you did. I just responded that we're going to change the script anyway with the deterministic lnd and so we can update this later.